### PR TITLE
Expose the Gui class to the taipy package.

### PR DESCRIPTION
**Pull request by FabienLelaquais**
+ apply RefMan conventions.

### CODE REVIEW

File: taipy/__init__.py [hunk 1]
```diff
@@ -1,12 +1,35 @@
-"""Top-level package for Taipy Gui.
+"""Taipy Graphical User Interface generator.
```
Comment (dinhlongviolin1) [inline taipy/__init__.py]: Correct me if i am wrong but shouldnt this file be placed in taipy/gui/__init__.py

Comment (FabienLelaquais) [reply taipy/__init__.py]: You are absolutely right.
This file should be more or less empty (except for the new import of the Gui class).
I'll fix that.
taipy/gui/__init__,py has been augmented properly.

**Review by dinhlongviolin1 — APPROVED** [review]
Looks good to me!

File: taipy/gui/__init__.py [hunk 2]
```diff
@@ -1,15 +1,36 @@
...
+elements.
 
 Each page can contain regular text and images, as well
 as Taipy controls that are typically linked to some
 value that is managed by the whole Taipy application.
+
+Here is how you can create your first Taipy User Interface:
+
+   - Create a Python source file.
+     Copy these two lines into a file called _taipy.py_.
```
Comment (FredLL-Avaiga) [inline taipy/gui/__init__.py]: no name clash with taipy
Even if not, I think it's not clear enough
Maybe use taipy_app.py

File: taipy/gui/gui.py [hunk 3]
```diff
@@ -117,16 +130,16 @@ def __init__(
                 The default value is a file that has the same base name as the Python
                 file defining the `main` function, sitting next to this Python file,
                 with the `.css` extension.
-            path_mapping: TODO explain what this does.
-            env_filename: An optional file from which to load application configuration
-                variables (see the
+            path_mapping (Optional[dict]): TODO explain what this does.
```
Comment (FredLL-Avaiga) [inline taipy/gui/gui.py]: association between URL path and file system

File: taipy/gui/partial.py [hunk 4]
```diff
@@ -43,8 +43,8 @@ def update_content(self, state: State, content: str):
         """Update partial content.
 
         Arguments:
-            state: the current user state as received in any callback.
-            content: the new content to show.
+            state: The current user state as received in any callback.
```
Comment (FredLL-Avaiga) [inline taipy/gui/partial.py]: no (State^) here ?

**Review by FredLL-Avaiga — APPROVED** [review]
a few comments

File: taipy/gui/__init__.py (hunk 2 above)
Comment (FabienLelaquais) [reply taipy/gui/__init__.py]: Makes sense.

File: taipy/gui/gui.py (hunk 3 above)
Comment (FabienLelaquais) [reply taipy/gui/gui.py]: Let me give this a try.

File: taipy/gui/partial.py (hunk 4 above)
Comment (FabienLelaquais) [reply taipy/gui/partial.py]: There should be.

**Pull request merged**